src/components/AppNavigation/index.vue

```diff
@@ -0,0 +1,26 @@
+<template>
+    <nav class="app-navigation">
+        <ul class="app-navigation__list list">
+            <li class="app-navigation__item">
+                <a href="" class="link">Catalogue</a>
```

Comment:
It seems that you're not using the "link" class right now; if that's the case, you can just delete it for now. When you start using it, please ensure that this class follows the BEM methodology.